ksql-parser/src/main/antlr4/io/confluent/ksql/parser/SqlBase.g4

```diff
@@ -552,14 +517,6 @@ fragment LETTER
     : [A-Z]
     ;
 
-SIMPLE_COMMENT
```

Comment:
We do use comments. Please don't remove them.

Reply:
Oops! Good catch. I've added a couple tests to avoid this mistake in the future. Thanks.